### DISCUSSION TITLE
Tag to document a block of informations?

### DISCUSSION THREAD

**Discussion by Timbuktu1982** [EnterTheNameHere/esdoc-monorepo#11] [Ideas]
Hi there,

i think it would be good if we could document specific code blocks summarized as a whole (e.g. named as "@block").

Example:
```
/**
 * Service injections.
 * 
 * @block ServiceInjections
 */
@service exampleFirst;
@service exampleSecond;
@service exampleThird;
```

**Comment by EnterTheNameHere** [comment]
Is this from an existing code? Is it like a section which would group these together on a page?

**Comment by Timbuktu1982** [comment]
Yes, e.g. in emberjs (and other languages) it is a good idea to group such general things. Right now I'm documenting everything separately, but that's not so useful. And I thus have to document the same thing for many target objects, so I don't have such a good overview.

At the moment:
```
    /**
     * Inject design service.
     *
     * @access  public
     * @type    {object}
     */
    @service design;

    /**
     * Inject globalUtilitiesBrowser service.
     *
     * @access  public
     * @type    {object}
     */
    @service globalUtilitiesBrowser;

    /**
     * Inject showModalDialog service.
     *
     * @access  public
     * @type    {object}
     */
    @service showModalDialog;
```

Better is a shorter documentation, because developer can show in target to get more details if needed.

```
    /**
     * Services.
     */
    @service design;
    @service globalUtilitiesBrowser;
    @service showModalDialog;
```

**Comment by EnterTheNameHere** [comment]
I have to check what ESDoc does with decorators, or more specifically how Babel underneath parses them.
Are you using Babel or Typescript or Flow to transpile to JS code?

**Comment by Timbuktu1982** [comment]
Ember uses babel to build / transpile the final app...
